olca-app/src/org/openlca/app/util/Colors.java

```diff
@@ -164,6 +164,36 @@ public static Color darkBackground() {
 		return get(47, 47, 47);
 	}
 
+	public static Color greenDQ() {
```

Comment:
`Colors` is a utility class. Utility methods make sense when they are used more then once or in a central configuration point. The latter is probably the idea here. However, as they are only used in DQUI and it is easy to find them there I would really keep the color definitions in DQUI. Also, `infDQ` and `supDQ` are not so easy to understand in the `Colors` context.